collect_app/src/androidTest/java/org/odk/collect/android/feature/formentry/FormSavedSnackbarTest.kt

```diff
@@ -44,9 +44,8 @@ class FormSavedSnackbarTest {
             .assertText(R.string.form_saved)
             .clickOnString(R.string.view_form)
             .assertText("25")
-            .assertTextDoesNotExist(R.string.jump_to_beginning)
-            .assertTextDoesNotExist(R.string.jump_to_end)
-            .assertText(R.string.exit)
+            .assertText(R.string.jump_to_beginning)
```

Comment:
I was thinking about this but:
1. I wanted to quickly bring a fix for the issue.
2. I think it's better to have tests for the current state even if it's weird and then in October we can fix it instead of mocking the clock and having tests that verify what we expect to have in the future.

Reply:
Ok I don't have strong feelings about it so I added separate tests fo the grace period and after.

Reply:
Oh I like that even more!